rpc_jobs/multi_node_aio_leapfrog.yml

```diff
@@ -0,0 +1,108 @@
+- project:
+    name: 'Multi-Node-AIO-Leapfrog-Testing'
+    series:
+      - kilo-r11.1.18:
+          branch: r11.1.18
+          branches: "r11.1.18"
+          KIBANA_SELENIUM_BRANCH: "liberty-12.2"
+          SERIES_USER_VARS: |
+            tempest_test_sets: 'scenario defcore api heat_api smoke'
+          DEPLOY_TELEGRAF: "yes"
```

Comment:
Think I jumped the gun with my LGTM, because I suspect we're hitting the same question here as we have in https://github.com/rcbops/rpc-gating/pull/422.  Notably, `branch` should be what we're upgrading to and `UPGRADE_FROM_REF` should be what we're upgrading from.

Also, we already have leapfrog jobs in `multi_node_aio.yml`.  Should all these MNAIO jobs be consolidated into this one file?

Lastly, so it's crystal clear, we're doing a leapfrog from kilo->liberty here?  I'm a bit confused by the liberty KIBANA_SELENIUM_BRANCH.

Reply:
Forget about consolidating leapfrog jobs, that's something we can do later.

Reply:
Pushed new rev up based on the recommendations and fixes on the AIO.